scene/main/canvas_item.h

```diff
@@ -161,6 +161,12 @@ class CanvasItem : public Node {
 
 	void _notification(int p_what);
 	static void _bind_methods();
+
+#ifndef DISABLE_DEPRECATED
+	void _draw_circle_bind_compat_84472(const Point2 &p_pos, real_t p_radius, const Color &p_color);
```

Comment:
```suggestion
	void _draw_circle_compat_84472(const Point2 &p_pos, real_t p_radius, const Color &p_color);
```
The `bind` part isn't needed here, just the normal method with `_compat_84472` suffixed